Core/clim-core/theming.lisp

```diff
@@ -0,0 +1,340 @@
+;;; ---------------------------------------------------------------------------
+;;;     Title: Themes
+;;;   Created: 2020-06-26 15:00
+;;;    Author: Daniel Kochmański <daniel@turtleware.eu>
+;;;   License: LGPL-2.1+ (See file 'Copyright' for details).
+;;; ---------------------------------------------------------------------------
+;;;
+;;;  (c) copyright 2001 by Michael McDonald <mikemac@mikemac.com>
+;;;  (c) copyright 2001 by Gilbert Baumann <unk6@rz.uni-karlsruhe.de>
+;;;  (c) copyright 2020 by Daniel Kochmański <daniel@turtleware.eu>
+;;;
+;;; ---------------------------------------------------------------------------
+;;;
+;;; Theming is a frequently requested feature. For now we'll just move
+;;; things related to theming to a single file so it is easier to come
+;;; up with the unified abstraction later on.
+;;;
```

Comment:
This follows the convention estabilished by Gilberth in soem files. IMO it makes sense to show, when the file was created and by whom. As of the title and notes below copyrights, it is also justified, because it is the best place to put todos (and in clim source code it is often done so), so it is the commiter responsibility to update comments. getting out of sync is of course possible, but I'd see that rather as an issue.